File: api-docs/packages/litro-laugfs-lpg-docs/python/smoke.py
"""Quick client smoke — polite delay on."""

from __future__ import annotations

import json

from litro_laugfs_lpg_docs import LitroLaugfsLpgDocsClient


def json_preview(data: object, limit: int = 200) -> str:
    try:
        return json.dumps(data, ensure_ascii=False)[:limit]
    except TypeError:
        return str(data)[:limit]


with LitroLaugfsLpgDocsClient(default_delay_seconds=1.0) as client:
    print("smoke", client.slug, "->", "litro_prices")
    data = client.litro_prices()
    preview = data[:200] if isinstance(data, str) else json_preview(data)
    print("ok", preview)

File: api-docs/packages/litro-laugfs-lpg-docs/javascript/examples/smoke.mjs
import { LitroLaugfsLpgDocsClient } from "../client.mjs";

const client = new LitroLaugfsLpgDocsClient({ defaultDelayMs: 1000 });
console.log("smoke", LitroLaugfsLpgDocsClient.slug, "->", "litroPrices");
const data = await client.litroPrices();
const preview = typeof data === "string" ? data.slice(0, 200) : JSON.stringify(data)?.slice(0, 200);
console.log("ok", preview);
